src/systemtest/java/reposense/ConfigSystemTest.java

```diff
@@ -75,11 +75,22 @@ public void test30DaysFromUntilDate() throws Exception {
      */
     @Test
     public void testDateRange() throws Exception {
-        generateReport(getInputWithDates("1/9/2017", "30/10/2017"));
+        generateReport(getInputWithDates("1/9/2017", "30/10/2017"), false);
         Path actualFiles = loadResource(getClass(), "dateRange/expected");
         verifyAllJson(actualFiles, FT_TEMP_DIR);
     }
 
+    /**
+     * System test with a specified since date and until date, with the last modified date time in each
+     * line of code.
+     */
+    @Test
+    public void testDateRangeWithModifiedDateTimeInLines() throws Exception {
```

Comment:
Added another system test for this functionality, however, I feel that this is redundant and takes up space. Should I merge this test with the `testDateRange` as one?